### PR TITLE
Subscriptions settings fragment

### PR DESCRIPTION
**Pull request by zsofinnemeth**
activities instead of fragments.
I added buttons that navigate back to the main activity because the default android back button (of the phone) exits the app

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
                 <action android:name="android.intent.action.MAIN" />
+
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".activity.SettingsActivity"/>
+        <activity android:name=".activity.ManageSubscriptionsActivity"/>
     </application>
 
 </manifest>
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @zsofinnemeth empty line

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 2]
```diff
@@ -105,6 +102,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 break;
             case R.id.logout:
                 logOut();
+                break;
+            case R.id.manage_subscriptions:
+                Intent i = new Intent(this, ManageSubscriptionsActivity.class);
+                startActivity(i);
+                finish();
+                break;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @zsofinnemeth dont finish the activity

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 3]
```diff
@@ -105,6 +102,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+            case R.id.settings:
+                Intent j = new Intent(this, SettingsActivity.class);
+                startActivity(j);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: dont finish it here as well

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 4]
```diff
@@ -105,6 +102,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 break;
             case R.id.logout:
                 logOut();
+                break;
+            case R.id.manage_subscriptions:
+                Intent i = new Intent(this, ManageSubscriptionsActivity.class);
+                startActivity(i);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @zsofinnemeth you dont need the intent anymore, just instantuate it in the parameters

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/ManageSubscriptionsActivity.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        setContentView(R.layout.activity_manage_subscriptions);
+
+        TextView subscriptions = (TextView) findViewById(R.id.subscriptionsTextView);
+
+
+
+        Toolbar myToolbar = (Toolbar) findViewById(R.id.my_toolbar);
+        setSupportActionBar(myToolbar);
+
+        Button backButton = (Button) findViewById(R.id.subscriptionBack);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/ManageSubscriptionsActivity.java]: @zsofinnemeth button should be in the toolbar as a back icon

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/ManageSubscriptionsActivity.java (hunk 5 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/ManageSubscriptionsActivity.java]: added a backlog for myself to trello

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java (hunk 3 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: OK

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java (hunk 2 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: OK

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java (hunk 4 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: OK

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (zsofinnemeth) [reply app/src/main/AndroidManifest.xml]: OK

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/ManageSubscriptionsActivity.java (hunk 5 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/ManageSubscriptionsActivity.java]: @adamgyulavari it is done too!

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**